src/abp-filter-parser.js

```diff
@@ -585,7 +594,7 @@ export function matches(parserData, input, contextParams = {}, cachedInputData =
     // Check for exceptions only when there's a match because matches are
     // rare compared to the volume of checks
     let exceptionBloomFilterMiss = parserData.exceptionBloomFilter && !parserData.exceptionBloomFilter.substringExists(cleanedInput, fingerprintSize);
-    if (!exceptionBloomFilterMiss || hasMatchingFilters(parserData.exceptionFilters, parserData, input, contextParams, cachedInputData)) {
+    if (!exceptionBloomFilterMiss && hasMatchingFilters(parserData.exceptionFilters, parserData, input, contextParams, cachedInputData)) {
```

Comment:
Question: why the switch from `||` to `&&` here?

Reply:
[EDIT] Just found it in your technical design template, I see there is a link in there to the updated Brave version of this, do you also maybe want to point at this line from that doc as well? Would be helpful as a cross-reference.

Reply:
👍